src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SdkContentTests.cs

```diff
@@ -46,10 +47,32 @@ private void WriteTarballFileList(string? tarballPath, string outputFileName, bo
         fileListing = BaselineHelper.RemoveRids(fileListing, isPortable);
         fileListing = BaselineHelper.RemoveVersions(fileListing);
         IEnumerable<string> files = fileListing.Split(Environment.NewLine).OrderBy(path => path);
+        files = RemoveExclusions(
+                    files,
+                    GetExclusionFilters(
+                        Path.Combine(BaselineHelper.GetAssetsDirectory(), "SdkDiffExclusions.txt"),
+                        sdkType));
 
         File.WriteAllLines(outputFileName, files);
     }
 
+        private static IEnumerable<string> RemoveExclusions(IEnumerable<string> files, IEnumerable<string> exclusions) =>
+            files.Where(item => !exclusions.Any(p => FileSystemName.MatchesSimpleExpression(p, item)));
+
+        private static IEnumerable<string> GetExclusionFilters(string exceptionFilePath, string sdkType)
```

Comment:
Consider renaming `exceptionFilePath` to `exclusionsFilePath` for consistency.

Reply:
Fixed with https://github.com/dotnet/installer/pull/15650/commits/93d517d066b8bc06c876f50caa6bf84bb5c00d3d